core/controllers/base.py

```diff
@@ -93,28 +93,18 @@ class UserFacingExceptions(python_utils.OBJECT):
     class NotLoggedInException(Exception):
         """Error class for users that are not logged in (error code 401)."""
 
-        pass
```

Comment:
Any reason to get rid of the `pass`? It seems nice to have an explicit signal that the class is empty.

Reply:
I think it was a pyling problem, but I decided to ignore this pylint check for now but forgot to readd the `pass`es here. Done!